www/docs/en/dev/config_ref/index.md

```diff
@@ -39,222 +39,357 @@ Note that before version 3.3.1-0.2.0, the file existed at `app/www/config.xml`,
-  namely `versionCode` for Android, `CFBundleVersion` for iOS, and 
-  `packageVersion` for Windows. See the Additional Versioning section below 
-  for details.
+    <!-- iOS -->
+    <widget CFBundleVersion="0.1.3">
+    </widget>
 
-- The `<name>` element specifies the app's formal name, as it appears
-  on the device's home screen and within app-store interfaces.
+    <!-- Windows -->
+    <widget packageVersion="0.1.3">
+    </widget>
 
-- The `<description>` and `<author>` elements specify metadata and
-  contact information that may appear within app-store listings.
 
-- The optional `<content>` element defines the app's starting
-  page in the top-level web assets directory. The default value is
-  `index.html`, which customarily appears in a project's top-level
-  `www` directory.
+## name
+   Specifies the app's formal name, as it appears on the device's home screen and within app-store interfaces.
 
-- `<access>` elements define the set of external domains the app is
-  allowed to communicate with. The default value shown above allows
-  it to access any server. See the Domain [Whitelist Guide](../guide/appdev/whitelist/index.html) for details.
+   Examples:
 
-- The `<preference>` tag sets various options as pairs of
-  `name`/`value` attributes. Each preference's `name` is
-  case-insensitive.  Many preferences are unique to specific
-  platforms, as listed at the top of this page. The following sections
-  detail preferences that apply to more than one platform.
+    <widget ...>
+        <name>HelloCordova</name>
+    </widget>
+    
+## description
+   Specifies metadata that may appear within app-store listings.
 
-### Additional Versioning
+   Examples:
+   
+    <widget ...>
+        <description>A sample Apache Cordova application</description>
+    </widget>
 
-Android, iOS, and Windows support a second version string (or number) in 
-addition to the one visible in app stores:
+    
+## author
+   Specifies contact information that may appear within app-store lisitngs.
+   
+   Attributes(type) | Description
+   ----------------- | ------------
+   email(string) | *Required* <br/> Email of the author.
+   href(string) | *Required* <br/> Website of the author.
 
-- [versionCode](http://developer.android.com/tools/publishing/versioning.html)
-  for Android
-- [CFBundleVersion](http://stackoverflow.com/questions/4933093/cfbundleversion-in-the-info-plist-upload-error)
-  for iOS
-- [packageVersion](https://msdn.microsoft.com/en-us/library/windows/apps/br211441.aspx)
-  for Windows
+   Examples:
 
-Below is an example that explicitly sets these properties:
+    <widget ...>
+        <author email="dev@cordova.apache.org" href="http://cordova.io"></author>
+    </widget>
 
-        <widget id="io.cordova.hellocordova"
-          version="0.0.1"
-          android-versionCode="7"
-          ios-CFBundleVersion="3.3.3"
-          windows-packageVersion="1.0.1.2">
+    
+## content
+   Optional.
```

Comment:
We should probably remove this since we are not specifying these for other optional elements.